### PR TITLE
feat(dashboard): change language code and use formatted date for widgets

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

image

### CODE REVIEW

File: apps/web/src/common/composables/date-range-formatter/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        if (state.isStartEndTheSameYear) {
+            if (state.isStartEndTheSameMonth) {
+                return `${dateFormatter(start, 'MMM')} ${dateFormatter(start, 'YYYY')}`;
+            }
+
+            return `${dateFormatter(start, 'MMM')} ~ ${dateFormatter(end, 'MMM')} ${dateFormatter(end, 'YYYY')}`;
+        }
+
+        return `${dateFormatter(start, 'MMM')} ${dateFormatter(start, 'YYYY')} ~ ${dateFormatter(end, 'MMM')} ${dateFormatter(end, 'YYYY')}`;
+    });
```
Comment (sulmoJ) [inline apps/web/src/common/composables/date-range-formatter/index.ts:43]: Praise: Coooooool

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: apps/web/src/services/dashboards/widgets/cost-widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue [hunk 2]
```diff
@@ -234,6 +235,11 @@ const state = reactive({
 
 const widgetFrameProps:ComputedRef = useWidgetFrameProps(props, state);
 
+const [totalSpentPeriod] = useDateRangeFormatter({
+    start: computed(() => state.settings?.date_range?.start),
+    end: computed(() => state.settings?.date_range?.end),
+});
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/cost-widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue:241]: praise: 🥰

**Review by yuda110 — APPROVED** [review]
wow!!! LGTM!

**Pull request merged**